### PR TITLE
sync with ardupilot board IDs

### PR DESCRIPTION
**Pull request by tridge**

### CODE REVIEW

File: board_types.txt [hunk 1]
```diff
@@ -51,9 +51,15 @@ AP_HW_F35LIGHTNING                    135
 AP_HW_MRO_X2V1_777                    136
 AP_HW_OMNIBUSF4V6                     137
 AP_HW_HELIOSPRING                     138
-AP_HW_PIXHAWK4PRO                     139
+AP_HW_PIXHAWK6                        139
```
Comment (davids5) [inline board_types.txt]: @tridge - I am not sure this should be called  PIXHAWK6 as it is not a FMUv6. 

FYI @LorenzMeier

Comment (tridge) [reply board_types.txt]: the name is chosen by the manufacturer, Holybro. If Lorenz wants to object he should take it up with Holybro

Comment (tridge) [reply board_types.txt]: the name has now been changed to Durandal by Holybro

Comment (davids5) [reply board_types.txt]: @tridge - thank you!

**Comment by davids5** [comment]
@pkocmoud - Thank you!

**Pull request merged**